helpers/crud/configuracion/configuracion.go

```diff
@@ -0,0 +1,55 @@
+package configuracion
+
+import (
+	"net/url"
+
+	"github.com/astaxie/beego"
+	models_configuracion "github.com/udistrital/configuracion_api/models"
+	"github.com/udistrital/utils_oas/request"
+)
+
+func ObtenerProceso(sigla string) (configuracion []models_configuracion.Proceso, outputError map[string]interface{}) {
+	defer func() {
+		if err := recover(); err != nil {
+			outputError = map[string]interface{}{
+				"funcion": "ObtenerProceso - Unhandled Error!",
+				"err":     err,
+				"status":  "500",
+			}
+			panic(outputError)
+		}
+	}()
+	parametros := url.Values{}
+	parametros.Add("query", "Sigla_contains:"+sigla)
+	urlproceso := beego.AppConfig.String("configuracionCrudService") + "proceso?" + parametros.Encode()
+	if err := request.GetJson(urlproceso, &configuracion); err != nil {
+		return configuracion, map[string]interface{}{
+			"funcion": "ObtenerProceso - request.GetJson(urlproceso, &configuracion)",
+			"err":     err,
+			"status":  "500",
```

Comment:
(En general) Usar el [HTTP 502](https://developer.mozilla.org/es/docs/Web/HTTP/Status/502), que es como lo mas parecido a errores que no dependan del usuario de la API sino que obedezcan a factores externos. Dejar el [HTTP 500](https://developer.mozilla.org/es/docs/Web/HTTP/Status/500) para los `defer func()`
```suggestion
			"status":  "502",
```